Master-Game/Entities/BaseEntity.cs

```diff
@@ -31,5 +33,15 @@ public bool isAlive()
         {
             return HealthPoints > 0.0f;
         }
+
+        public void AddDamage(float appliedDamage)
```

Comment:
This function is doing two things. It's adding damage and removing it. Make two separate function. ApplyDamage and ApplyHealth

Reply:
May make it one function instead?